apps/financial-aid/web-osk/src/components/Login/login.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { GridContainer, Button } from '@island.is/island-ui/core'
+
+import { useRouter } from 'next/router'
+
+const Login = () => {
+  const router = useRouter()
+
+  return (
+    <GridContainer>
+      <Button
+        onClick={() => {
+          router.push('/api/auth/login')
```

Comment:
Do we want to have this route as a constant somewhere? In this file we are using it in three different places, so to me it feels like it is time to have it as a constant 😄